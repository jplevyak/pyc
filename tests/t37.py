a = (1, "asdf", 2.0)
a = (2, "fdsa", 3.0)
print a[0]
print a[1]
print a[2]
