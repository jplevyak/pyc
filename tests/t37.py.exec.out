2
fdsa
3.0
